plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3select/TestS3SelectPushdown.java

```diff
@@ -13,34 +13,204 @@
  */
 package io.trino.plugin.hive.s3select;
 
+import io.trino.plugin.hive.HiveStorageFormat;
+import io.trino.plugin.hive.HiveType;
+import io.trino.plugin.hive.metastore.Column;
+import io.trino.plugin.hive.metastore.Partition;
+import io.trino.plugin.hive.metastore.Storage;
+import io.trino.plugin.hive.metastore.StorageFormat;
+import io.trino.plugin.hive.metastore.Table;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.testing.TestingConnectorSession;
 import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe;
 import org.apache.hadoop.mapred.JobConf;
 import org.apache.hadoop.mapred.TextInputFormat;
+import org.apache.hive.hcatalog.data.JsonSerDe;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.OptionalLong;
+import java.util.Properties;
+
 import static io.trino.hadoop.ConfigurationInstantiator.newEmptyConfiguration;
+import static io.trino.plugin.hive.HiveMetadata.SKIP_FOOTER_COUNT_KEY;
+import static io.trino.plugin.hive.HiveMetadata.SKIP_HEADER_COUNT_KEY;
+import static io.trino.plugin.hive.HiveType.HIVE_BINARY;
+import static io.trino.plugin.hive.s3select.S3SelectPushdown.isCompressionCodecSupported;
+import static io.trino.plugin.hive.s3select.S3SelectPushdown.isSplittable;
+import static io.trino.plugin.hive.s3select.S3SelectPushdown.shouldEnablePushdownForTable;
+import static io.trino.spi.session.PropertyMetadata.booleanProperty;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonList;
+import static org.apache.hadoop.hive.serde.serdeConstants.SERIALIZATION_LIB;
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
 public class TestS3SelectPushdown
 {
     private TextInputFormat inputFormat;
+    private ConnectorSession session;
+    private Table table;
+    private Partition partition;
+    private Storage storage;
+    private Column column;
+    private Properties schema;
+    private static final String S3_SELECT_PUSHDOWN_ENABLED = "s3_select_pushdown_enabled";
```

Comment:
nit: move it before the class instance fields (usually we are trying to keep the static constants first). Separate them with an empty line